docker-compose.yml

```diff
@@ -108,6 +108,49 @@ services:
     image: "ghcr.io/nasa-ammos/aerie-ui:develop"
     ports: ["80:80"]
     restart: always
+  merlin_worker_1:
+    build:
+      context: ./merlin-worker
+      dockerfile: Dockerfile
+    container_name: aerie_merlin_worker_1
+    depends_on: ["postgres"]
+    environment:
+      MERLIN_WORKER_DB: "aerie_merlin"
+      MERLIN_WORKER_DB_PASSWORD: "aerie"
+      MERLIN_WORKER_DB_PORT: 5432
+      MERLIN_WORKER_DB_SERVER: "postgres"
+      MERLIN_WORKER_DB_USER: "aerie"
+      MERLIN_WORKER_LOCAL_STORE: /usr/src/app/merlin_file_store
+      JAVA_OPTS: >
+        -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005
+        -Dorg.slf4j.simpleLogger.defaultLogLevel=DEBUG
+        -Dorg.slf4j.simpleLogger.logFile=System.err
+    ports: ["5007:5005"]
+    restart: always
+    volumes:
+      - aerie_file_store:/usr/src/app/merlin_file_store:ro
+  merlin_worker_2:
+    build:
+      context: ./merlin-worker
+      dockerfile: Dockerfile
+    container_name: aerie_merlin_worker_2
+    depends_on: ["postgres"]
+    environment:
+      MERLIN_WORKER_DB: "aerie_merlin"
+      MERLIN_WORKER_DB_PASSWORD: "aerie"
+      MERLIN_WORKER_DB_PORT: 5432
+      MERLIN_WORKER_DB_SERVER: "postgres"
+      MERLIN_WORKER_DB_USER: "aerie"
+      MERLIN_WORKER_LOCAL_STORE: /usr/src/app/merlin_file_store
+      MERLIN_WORKER_PORT: 27186
```

Comment:
🔵 I'm curious about this port - what is it used for? Should we expose it?

Reply:
Good catch. A rogue copy and paste.